app/models/metric_rollup.rb

```diff
@@ -4,7 +4,7 @@ class MetricRollup < ApplicationRecord
   self.primary_key = "id"
 
   include Metric::Common
-  include_concern 'Metric::ChargebackHelper'
+  include Metric::ChargebackHelper
```

Comment:
Here's one that's different.  `ChargebackHelper` doesn't live within the current namespace, `MetricRollup`, so we need to autoload via `Metric::ChargebackHelper` so it can be found.